flexiblepricing/management/commands/create_product_for_program_courses.py

```diff
@@ -0,0 +1,85 @@
+from django.contrib.contenttypes.models import ContentType
+from django.core.management import BaseCommand, CommandError
+
+from courses.models import CourseRun, Program
+from ecommerce.models import Product
+
+
+class Command(BaseCommand):
+    """
+    Create products for all the course runs in a program with standard price.
+    The default behavior is to create "inactive" product for each of DEDP courses with price $1000.
+    Unless specifying --active, running it more than once with default argument will create duplicated 'inactive' products
```

Comment:
I am not aware of a way to avoid creating a duplicate inactive production if there already exist. This is done at the DB level, in particular, this is how it's done in ProductsQuerySet. It's the same thing if create inactive product in django admin. There shouldn't be side effect of creating inactive products
I tested a few times for this, but if you know a way to avoid it, please let me know

Reply:
There should only be one active product for a given courseware object (as noted, this is enforced in the code). Multiple inactive products can exist, though, because we may want to have a different product for special pricing on something, or we may want to ID people differently based on product for the same course run, etc.